packages/app/src/__deprecated__/renderer/locales/default/en-US.json

```diff
@@ -420,6 +419,28 @@
   "module.messages.sendButtonTooltipDescription": "Send message",
   "module.messages.textAreaPlaceholder": "Type your message",
   "module.messages.unreadOnly": "Unread only",
+
+  "module.messages.deletedThread": "1 conversation was deleted.",
```

Comment:
The messages that were previously used and now are not should be removed (e.g. `module.messages.deletingModalTitle` or `module.messages.deleteMessageConfirmationModalBody`).

Reply:
https://github.com/mudita/mudita-center/pull/827/commits/4efd4e16515f625821527dcfda069a6477530456